CrankShaft/code/hubl/theme-variables.css

```diff
@@ -14,6 +14,34 @@
 {% set accentColor2   = "#ccc" %}    /* {# Light Gray  |  Ex. "color: {{ accentColor2 }};" #} */
 {% set accentColor3   = "#ddd" %}    /* {# Lightest Gray  |  Ex. "color: {{ accentColor3 }};" #} */
 
+
+/* ======  @D.holdt addressing Add Color Variables to theme-variables.css ======*/
+/* How to use Color Variables 
+*
+*  1. Define your color variables semantically in {# what is the appropriate file for proper import? #} {# Ex. {% set brandBlue = #0366d6 %}
```

Comment:
To answer the question on what's proper file to import.

This theme-variables file will be imported into all of the other files. So it's vars are basically global through the framework.